build/lib/n2y/converter.py

```diff
@@ -0,0 +1,473 @@
+import importlib.util
```

Comment:
@sHermanGriffiths I don't think the `build` directory should be included in this PR. Please review all of your files before committing them. Also, I recommend against using the `-a` command with `git commit` since it makes it too easy to accidentally add files like this. Also, feel free to update the `.gitignore` file as needed.